app/code/Magento/InventoryIndexer/composer.json

```diff
@@ -9,7 +9,8 @@
         "magento/module-inventory-api": "*",
         "magento/module-inventory-catalog-api": "*",
         "magento/module-inventory-sales-api": "*",
-        "magento/module-inventory-sales": "*"
+        "magento/module-inventory-sales": "*",
+        "magento/module-catalog": "*"
```

Comment:
Can you please make dependency on `magento/module-catalog": `
suggested in both modules 
InventoryCache and InventoryIndexer